Sources/Risk/Services/RiskError.swift

```diff
@@ -1,35 +1,52 @@
 //
 //  RiskError.swift
-//  
+//
 //
 //  Created by Precious Ossai on 13/10/2023.
 //
 
 import Foundation
 
 public enum RiskError: LocalizedError, Equatable {
```

Comment:
btw, this doesn't need to conform to `LocalizedError` anymore I guess. Possibly not to `Equatable` as well. Can be tried to minimise the conformances.